### PR TITLE
Capstone project - Science Festival Site

### PR DESCRIPTION
**Pull request by jaflih**
### Capstone project - Science Festival Site

- Added HTML/CSS/JS linters
- Added Fonts : Cocogoose & Lato
- Added SASS environment: CSS dynamically generated 
- Added HTML files :  **homepage and about page**
- Added Mobile and Desktop view
- Added Open / Close Menu in mobile view
- Added "Speakers" section generated dynamically
- Added 2 styles for the mobile menu: background transparently or white
- Added feature: Randomly generate speaker data using 2 external libraries (faker and randomuser)

Link to my [video](https://www.loom.com/share/85a9696a71a745519d2e86975763d119)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Science Festival Site</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <!-- HEADER -->
+    <header class="header">
```
Comment (elyor-doniyorov) [inline about.html:12]: - [ ] Make sure the desktop version is divided into a top dark bar and a bottom white bar for both `home `and `about `pages.
Also, the top short bar section with social media links, logout, and language links is aligned to the right.
Let's compare and fix them: 👇 
- Your design: 👇 
homepage-1
- Original: 👇 
homepage+1_LI

File: about.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <div class="event_container">
+          <div class="orange"><span class="date">2020</span><span class="description">Science festival in Grenoble</span></div>
+          <img src="assets/images/events/2020.jpg" class="event_image" alt="Image for the 2020 event" />
+        </div>
+      </div>
+    </section>
+    <!-- EVENTS END -->
+
+    <!-- FOOTER -->
+    <footer>
```
Comment (elyor-doniyorov) [inline about.html:98]: - [ ] According to the requirements the footer has a white background on mobile size and a dark background on desktop size. Kindly fix it! 🙏

File: about.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum consequuntur ducimus illum ullam laudantium consequatur quis expedita
+        veritatis architecto aliquid mollitia commodi nisi enim reprehenderit voluptates non facere, quaerat iste?
+      </p>
+      <img src="assets/images/logo_blanc.png" class="commons_logo" alt="Science Festival Logo" />
+    </section>
+    <!-- COMMONS END -->
+
+    <!-- EVENTS -->
+    <section class="events">
+      <h2>See the past Science Festival events</h2>
```
Comment (elyor-doniyorov) [inline about.html:81]: - [ ] Make sure you follow [the design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in this section, such as regarding **_Layout: composition and space between elements_**. Let's compare your design and the original below 👇 
- Yours here: 👇 
seepast-1_LI
- Original here: 👇 
seepast+1_LI
Kindly fix it 🙏

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @jihaneH, 👋

You did an awesome job, I like it! 🙌
However, there are some issues that you still need to work on to go to the next project but you are almost there!
almost-there


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (jaflih) [reply about.html:12]: It's done

Capture d’écran 2021-12-21 à 19 33 51

File: about.html (hunk 2 above)
Comment (jaflih) [reply about.html:98]: it's done
Capture d’écran 2021-12-21 à 19 34 12
Capture d’écran 2021-12-21 à 19 34 27

File: about.html (hunk 3 above)
Comment (jaflih) [reply about.html:81]: it's done
Capture d’écran 2021-12-21 à 19 42 00

**Review by leolpaz — APPROVED** [review]
Hi @jihaneH , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing all the changes required by the previous reviewer

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaflih** [comment]
icegif-1433

**Pull request merged**